packages/image_picker/android/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove eclipse artifacts from the repo (`.classpath`, `.project`, `.settings/`).